cli/utilities/utils.py

```diff
@@ -520,3 +520,40 @@ def get_disk_devlinks(node, disk):
         elif "by-path" in path:
             by_path.append(path)
     return by_id, by_path
+
+
+def podman_login(node, registry, username, password):
+    """
+    Podman Login
+    Args:
+        node (ceph): Node to execute cmd
+        registry (str): registry name e.g: registry.redhat.io
+        username (str): registry user name
+        password (str): registry password
+    Return:
+
+    """
+    cmd = f"podman login -u {username} -p {password} {registry}"
+    return node.exec_command(sudo=True, cmd=cmd)
+
+
+def podman_pull_image(node, image):
```

Comment:
As discussed move this methods to package - https://github.com/red-hat-storage/cephci/blob/master/cli/utilities/containers.py